serverlessworkflow/modules/ROOT/pages/supporting-services/jobs-service.adoc

```diff
@@ -0,0 +1,107 @@
+= Job Service
+:compat-mode!:
+// Metadata:
+:description: Job Service to control timeouts in {context}
+:keywords: kogito, workflow, serverless, timeout, timer, expiration, job service
+// links
+:jobs_service_image_url: https://quay.io/repository/kiegroup/kogito-jobs-service-allinone
+:jobs_service_image_usage_url: https://github.com/kiegroup/kogito-images#jobs-services-all-in-one
+
+In {product_name} architecture there is a dedicated supporting service that is responsible for controlling the execution of
+time-triggered actions, also known as jobs.
+
+The job execution is a callback to the {context} runtime application. This callback can be configured in different ways as described in the <<job-service-communication, communication>> section.
+
+[#job-service]
+== Job Service configuration
+
+All timer nodes that might be declared in a workflow, are handled by the Job Service, which is responsible for managing, scheduling, and firing all actions (jobs) to be executed in the workflows.
+
+[NOTE]
+====
+Suppose the workflow service is not configured to use job service or there is no such service running.
+In that case, all timer-related actions use an embedded in-memory implementation of job service, which should not be used in production, since when the application shutdown, all timers are lost, which in a serverless architecture is a very common behavior with the scale to zero approach.
```

Comment:
```suggestion
In that case, all timer-related actions use an embedded in-memory implementation of the job service, which should not be used in production, since when the application shutdown, all timers are lost, which is a serverless architecture is a very common behavior with the scale to zero approaches.
```